honeybee_radiance/config.py

```diff
@@ -0,0 +1,308 @@
+        if cfg is None:
+            cfg = os.path.join(os.path.dirname(__file__), 'config.json')
+        self._load_from_file(cfg)
+        self._config_file = cfg
+
+    def _load_from_file(self, file_path):
+        """Set all of the the properties of this object from a config JSON file.
+        
+        Args:
+            file_path: Path to a JSON file containing the file paths. A sample of this
+                JSON is the config.json file within this package.
+        """
+        # check the default file path
+        assert os.path.isfile(str(file_path)), \
+            ValueError('No file found at {}'.format(file_path))
+
+        # set the default paths to be all blank
+        default_path = {
+            "radiance_path": r'',
+            "standards_data_folder": r''
+        }
+
+        with open(file_path, 'r') as cfg:
+            try:
+                paths = json.load(cfg)
+            except Exception as e:
+                print('Failed to load paths from {}.\n{}'.format(file_path, e))
+            else:
+                for key, p in paths.items():
+                    if not key.startswith('__') and p.strip():
+                        default_path[key] = p.strip()
+
+        # set path for radiance installations
+        self.radiance_path = default_path["radiance_path"]
+
+        # set path for the standards_data_folder
+        self.standards_data_folder = default_path["standards_data_folder"]
+
+    def _find_radiance_folder(self):
+        """Find the Radiance installation in its default location.
+        
+        This method will first attempt to return the path of a standalone Radiance
+        installation and, if none are found, it will search for one that is
+        installed with OpenStudio.
+
+        Returns:
+            File directory and full path to executable in case of success.
+            None, None in case of failure.
+        """
+        # first check for the default location where standalone Radiance is installed
+        rad_path = None
+        if os.name == 'nt':  # search the C:/ drive on Windows
+            for f in os.listdir('C:\\'):
+                if f.lower() == 'radiance' and os.path.isdir('C:\\{}'.format(f)):
+                    rad_path = 'C:\\{}'.format(f)
+                    break
+        elif platform.system() == 'Darwin':  # search the Applications folder on Mac
+            for f in os.listdir('/Applications/'):
+                if f.lower() == 'radiance' and os.path.isdir('/Applications/{}'.format(f)):
+                    rad_path = '/Applications/{}'.format(f)
+                    break
+        elif platform.system() == 'Linux':  # search the usr/local folder
+            for f in os.listdir('/usr/local/'):
+                if f.lower() == 'radiance' and os.path.isdir('/usr/local/{}'.format(f)):
+                    rad_path = '/usr/local/{}'.format(f)
+                    break
+        else:  # then check the Radiance that comes with OpenStudio
```

Comment:
Should the check here be be `if rad_paths is None`? Right now you will probably never hit this condition.

Reply:
Yes, I realized this error in my sleep last night. I'll fix it now.